<!DOCTYPE html>
<html>









  <head>
    <meta charset="utf-8">
    <title>Introduction to proteomics, protein identification, quantification and statistical modelling</title>
    
        <script async defer data-domain="training.galaxyproject.org" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>

    
    <link rel="stylesheet" href="/training-material/assets/css/slides.css">
    <script src="https://kit.fontawesome.com/67b3f98409.js" crossorigin="anonymous"></script>
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <link rel="shortcut icon" href="/training-material/favicon.ico" type="image/x-icon" />

    
    
    
    
    <meta name="description" content="Training material for proteomics workflows in Galaxy" />
    <meta property="og:title" content="Galaxy Training: Introduction to proteomics, protein identification, quant..." />
    <meta property="og:description" content="Training material for proteomics workflows in Galaxy" />
    <meta property="og:image" content="/training-material/assets/images/GTNLogo1000.png" />
    <script type="application/ld+json">
      


{
  "@context": "http://schema.org",
  "@type": "Course",
  "accessMode": [
    "textual",
    "visual"
  ],
  "accessModeSufficient": [
    "textual",
    "visual"
  ],
  "accessibilityControl": [
    "fullKeyboardControl",
    "fullMouseControl"
  ],
  "accessibilityFeature": [
    "alternativeText",
    "tableOfContents"
  ],
  "accessibilitySummary": "Short descriptions are present but long descriptions will be needed for non-visual users",
  "audience": {
    "@type": "EducationalAudience",
    "educationalRole": "students"
  },
  "citation": {
    "@type": "CreativeWork",
    "name": "Community-Driven Data Analysis Training for Biology",
    "url": "https://doi.org/10.1016/j.cels.2018.05.012"
  },
  "copyrightHolder": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "discussionUrl": "https://gitter.im/Galaxy-Training-Network/Lobby",
  "headline": "Introduction to proteomics, protein identification, quantification and statistical modelling",
  "inLanguage": {
    "@type": "Language",
    "name": "English",
    "alternateName": "en"
  },
  "interactivityType": "mixed",
  "isAccessibleForFree": true,
  "license": "https://github.com/galaxyproject/training-material/blob/master/LICENSE.md",
  "producer": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "provider": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "sourceOrganization": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "isPartOf": {
    "@type": "CreativeWork",
    "name": "Proteomics",
    "description": "Training material for proteomics workflows in Galaxy",
    "url": "https://training.galaxyproject.org//training-material/topics/proteomics/"
  },
  "courseCode": "proteomics / introduction / introduction.html",
  "learningResourceType": "slides",
  "name": "Introduction to 'Proteomics'",
  "url": "https://training.galaxyproject.org//training-material/topics/proteomics/slides/introduction.html",
  "description": "Slides for Proteomics",
  "coursePrerequisites": [
    {
      "@type": "CreativeWork",
      "url": "https://training.galaxyproject.org//training-material/topics/introduction/",
      "name": "Introduction to Galaxy Analyses",
      "description": "Introduction to Galaxy Analyses",
      "provider": {
        "@type": "Organization",
        "email": "galaxytrainingnetwork@gmail.com",
        "name": "Galaxy Training Network",
        "url": "https://galaxyproject.org/teach/gtn/"
      }
    }
  ],
  "hasPart": [

  ],
  "author": [
    {
      "@type": "Person",
      "name": "Melanie Föll"
    }
  ],
  "contributor": [
    {
      "@type": "Person",
      "name": "Melanie Föll"
    }
  ],
  "about": [
    {
      "@type": "CreativeWork",
      "name": "Proteomics",
      "description": "Training material for proteomics workflows in Galaxy",
      "url": "https://training.galaxyproject.org//training-material/topics/proteomics/"
    },
    {
      "@type": "DefinedTerm",
      "@id": "http://edamontology.org/topic_0121",
      "inDefinedTermSet": "http://edamontology.org",
      "termCode": "topic_0121",
      "url": "https://bioportal.bioontology.org/ontologies/EDAM/?p=classes&conceptid=http%3A%2F%2Fedamontology.org%2Ftopic_0121"
    }
  ]
}
    </script>
    <script type="text/javascript" src="/training-material/assets/js/jquery.slim.min.js"></script>
  </head>
  <body>
    <textarea id="source">
name: inverse
layout: true
class: center, middle, inverse

<div class="my-header"><span>
<a href="/training-material/topics/proteomics" title="Return to topic page" ><i class="fa fa-level-up" aria-hidden="true"></i></a>

<a class="nav-link" href="https://github.com/galaxyproject/training-material/edit/master/topics/proteomics/slides/introduction.html"><i class="fa fa-pencil" aria-hidden="true"></i></a>

</span></div>

<div class="my-footer"><span>

<img src="/training-material/assets/images/GTN-60px.png" alt="Galaxy Training Network" style="height: 40px;"/>

</span></div>

---

<img src="/training-material/assets/images/GTN.png" alt="Galaxy Training Network" style="height: 8em;"/>

# Introduction to proteomics, protein identification, quantification and statistical modelling


<div class="contributors-line">


 <a href="/training-material/hall-of-fame/foellmelanie/" class="contributor-badge"><img src="https://avatars.githubusercontent.com/foellmelanie" alt="Melanie Föll">Melanie Föll</a>


</div>




---

## Requirements

Before diving into this slide deck, we recommend you to have a look at:



    
        
        
            
        
- [Introduction to Galaxy Analyses](/training-material/topics/introduction)
    






.footnote[Tip: press `P` to view the presenter notes]

???
Presenter notes contain extra information which might be useful if you
intend to use these slides for teaching.

Press `P` again to switch presenter notes off

Press `C` to create a new window where the same presentation will be displayed.
This window is linked to the main window. Changing slides on one will cause the
slide to change on the other. Useful when presenting.

---

## Introduction to proteomics, protein identification, quantification and statistical modelling

---

# Proteomics

*&quot;A Proteome is the entire protein complement expressed
by a genome or by a cell or tissue type at a given time.&quot;* - Marc Wilkins 1996

![Schematic overview of proteomics](/training-material/topics/proteomics/images/intro-slides/proteomics_schematic.png)

.footnote[Image credit: [Madprime](https://commons.wikimedia.org/wiki/File:Genetic_code.svg)]

---

# Mass Spectometry-based Proteomics



.force-right[
- Bottom-up approach: measurement of peptides after
enzymatic digestion of proteins
- Measures the peptides mass-to-charge (m/z) ratio and stores
m/z – intensity pairs in mass spectra
- High sensitivity and throughput
- Standard method for the analysis of complex samples
]

&lt;br&gt;&lt;br&gt;
![](/training-material/topics/proteomics/images/intro-slides/ms-proteomics.png)


---

# MS-based Proteomic Techniques


|   |   |   |
|:-:|:-:|:-:|
| .image-90[![](/training-material/topics/proteomics/images/intro-slides/explorative-proteomics.png)] | .image-90[![](/training-material/topics/proteomics/images/intro-slides/targeted-proteomics.png)] | .image-90[![](/training-material/topics/proteomics/images/intro-slides/ms-imaging.png)] |
| **Explorative Proteomics** | **Targeted Proteomics** | **Mass Spectometry** |
| **(DDA/DIA)** | **(SRM/PRM)** | **Imaging** |
| All proteins of a cell/organ(ism) | Focus on subset of proteins | Focus on location of proteins |

---

# Proteomics tools in Galaxy

![](/training-material/topics/proteomics/images/intro-slides/tools-overview.png)


---

# Active Development of Tools and Training Materials

---


# Explorative Proteomics


![](/training-material/topics/proteomics/images/intro-slides/explorative-proteomics.png)

**(DDA/DIA)**

All proteins of a cell / organ / organism

---

# Explorative Proteomics Workflow

![](/training-material/topics/proteomics/images/intro-slides/explorative-proteomics-workflow.png)

---

class: top

![](/training-material/topics/proteomics/images/intro-slides/wf1-sample-prep.png)

![](/training-material/topics/proteomics/images/intro-slides/sample-prep.png)


---
class: top

![](/training-material/topics/proteomics/images/intro-slides/wf2-mass-spec.png)

![](/training-material/topics/proteomics/images/intro-slides/mass-spec-schematic.png)


---
class: top

![](/training-material/topics/proteomics/images/intro-slides/wf2-mass-spec.png)

Tandem mass spectrometry (LC-MS/MS) using data dependent acquisition (Top3)

![](/training-material/topics/proteomics/images/intro-slides/mass-spec-schematic2.png)


---
class: top

![](/training-material/topics/proteomics/images/intro-slides/wf3-data-analysis.png)

![](/training-material/topics/proteomics/images/intro-slides/data-analysis-schematic.png)

---

# Typical label-free data dependent acquisition (DDA) workflow (tandem mass spectrometry)

![](/training-material/topics/proteomics/images/intro-slides/all-schematic.png)

![](/training-material/topics/proteomics/images/intro-slides/wf4-all.png)

---

# Peptide identification with MS2 fragment spectra

.left-column50[
![](/training-material/topics/proteomics/images/intro-slides/fragment-spectra.png)
]

.right-column40[
**Identification options:**

- Manual interpretation

- De novo sequencing

- Matching to in-silico spectra (generated from FASTA database)
]

---

# Peptide identification via an in-silico database

![](/training-material/topics/proteomics/images/intro-slides/insilico-identification1.png)

---

# Peptide identification via an in-silico database


![](/training-material/topics/proteomics/images/intro-slides/insilico-identification2.png)

---

# FASTA File Format

- Text based file to store DNA, RNA, protein sequences in a single letter code
- Each entry has
  - One header line: starting with `&gt;` and containing a unique identifier
  - A (nucleotide/amino acid) sequence


![](/training-material/topics/proteomics/images/intro-slides/fasta-proteomics.png)


---

# FASTA Files for Proteomics

- A proteome is the set of proteins thought to be expressed by an organism.


- Only proteins that are present in the FASTA file can be identified


- The more proteins are present in the FASTA file the higher the changes for false identifications
  - Choose the right FASTA file


- Source for proteome FASTA files:
  - Uniprot, neXtProt, NCBI, own/public DNA or RNA sequencing data



.footnote[
Tutorial: [Protein FASTA database handling](https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/database-handling/tutorial.html)
]

---

# Protein Identification via Protein Inference

![](/training-material/topics/proteomics/images/intro-slides/protein-interference.png)


---

# Quantification methods in proteomics


TODO Meli: Please add image here, it didn't look good on my off-brand powerpoint ;P

---


# Benefits and Drawbacks: Overview Table

.table-100[

| Category                  | Label-free   | Labelled |
|---------------------------|:------------:|:--------:|
| Machine Time              | More         | **Less** |
| Wet lab complexity &amp; Time &lt;span style=&quot;width:50%&quot;/&gt; | **Little**   | Medium   |
| Comparability of Samples  | Difficult    | **Easy** |
| Data Analysis             | Complex      | Complex  |
| Study Design              | **Flexible** | Fixed    |

]

Note: The **superior techniques** in each line is marked in **bold**


.footnote[ Tutorial: [Label-free versus Labelled: How to Choose Your Quantitation Method](https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/labelfree-vs-labelled/tutorial.html
) ]
---

# Quantification Methods in Proteomics

![](/training-material/topics/proteomics/images/intro-slides/quantification-methods.png)

---

# Label-free Peptide Quantification

![](/training-material/topics/proteomics/images/intro-slides/label-free-quantification.png)

---

# Protein Quantification

- **Potential Decisions:**
  - Include only quantifications of unique (proteotypic) peptides, or also „razor“ (ambiguous, assigned to the protein group with the most other peptides)
  - Include peptides with modifications?
  - How to compute protein abundance based on multiple peptides:
    - Median, Mean, Weighted Mean, Sum


.left-column60[
.image-90[![](/training-material/topics/proteomics/images/intro-slides/protein-quantification-table.png)]
]

.right-column40[

- **Fold Change:** Ratio of protein abundances in both samples.

- **Log2 Fold Change:** facilitates interpretation of changes
]

---

# MaxQuant Software


.left-column50[

.image-100[
  ![](/training-material/topics/proteomics/images/intro-slides/maxquant-paper.png)
]

- Freeware, &quot;Black Box&quot;
- Popular non-commercial proteomics software
]

.right-column50[

- **Raw data import**
- **Protein Identification:**
  - Andromeda Search Engine
  - Speciality: Match-between-runs
- **Protein Quantification:**
  - Label-free
  - Label-based
      - SILAC, Dimethyl, …
  - Reporter ion MS2
      - TMT, iTRAQ
]



.footnote[
Watch: [MaxQuant video on YouTube](https://www.youtube.com/channel/UCKYzYTm1cnmc0CFAMhxDO8w)
]

---

# Follow  up / Statistical Analysis

.pull-left[
**Typical follow up procedures:**


- Visualization
  - E.g. venn-diagramm, volcano plot, heatmap, boxplots, histogram


- Protein-Protein interaction network analysis

- GO annotation and enrichment analysis

- Differentially abundant protein analysis

]

.pull-right[
.image-100[
 ![](/training-material/topics/proteomics/images/intro-slides/statistical-analysis.png)
]
]

.footnote[
Müller, 2018, Neoplasia
]

---

#  MSstats Software

.left-column50[

.image-100[
  ![](/training-material/topics/proteomics/images/intro-slides/msstats-paper.png)
]

- Two Bioconductor R Packages
  - MSstats &amp; MSstatsTMT
- Popular open-source statistical proteomics software
]

.right-column50[

- **Import results from common proteomics software:**
  - MaxQuant, Skyline, OpenMS
- **Transformation and normalization**
- **Missing value imputation**
- **Protein summarization and quantification**
- **Statistical modelling**
  - linear models to detect differentially abundant proteins in label-free and isobaric labeled experiments

]



.footnote[
Watch: [MSstats videos on YouTube](https://www.youtube.com/c/MayInstituteNEU/search?query=msstats
)
]



---




## Related tutorials


    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    

    
        
    



---

## Thank you!

This material is the result of a collaborative work. Thanks to the [Galaxy Training Network](https://wiki.galaxyproject.org/Teach/GTN) and all the contributors!


<div class="contributors-line">


 <a href="/training-material/hall-of-fame/foellmelanie/" class="contributor-badge"><img src="https://avatars.githubusercontent.com/foellmelanie" alt="Melanie Föll">Melanie Föll</a>


</div>



<img src="/training-material/assets/images/GTN.png" alt="Galaxy Training Network" style="height: 100px;"/>


This material is licensed under the <a rel="license" href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.


    </textarea>
	<script src="/training-material/assets/js/remark-latest.min.js" type="text/javascript"></script>
    <script type="text/javascript" src="/training-material/assets/js/theme.js"></script>
    <script type="text/javascript" src="/training-material/assets/js/clipboard.min.js"></script>
    <script type="text/javascript">
      var slideshow = remark.create({navigation: {scroll: false,}});
      var hljs = remark.highlighter.engine;
      var snippets = document.querySelectorAll('code.remark-code');
        [].forEach.call(snippets,function(snippet){
          snippet.firstChild.insertAdjacentHTML('beforebegin','<button class="btn btn-light" data-clipboard-snippet><i class="fa fa-copy"></i>&nbsp;Copy</button>');
        });
      var clipboardSnippets=new ClipboardJS('[data-clipboard-snippet]',{
        target:function(trigger){return trigger.parentElement;
      }});
    </script>

    <script type="text/javascript">
        if(window.location.hostname === "galaxyproject.github.io") {
            // Redirect
            var redirect = "https://training.galaxyproject.org" + window.location.pathname + window.location.search;
            $('body').prepend("<div style='text-align: center'><strong>Note: </strong>This content has a new home at <a href=\"" + redirect + "\">" + redirect + "</a>, which you will be redirected to in 5 seconds.</div>");

            window.setTimeout(function(){
                window.location.href = redirect;
            }, 5000)

        }
    </script>

  </body>
</html>
